### ISSUE TITLE
Fix film and photography points

### ISSUE THREAD

**Issue by kelliedesigner** [barnardos/design-system#270]
## Description

Feedback from user suggests that 'in real locations' isn't clear, and that it is missing a point on having clear descriptions for screen readers.

## Step to reproduce

1. Go to Guidance
2. Film and Photography

**Comment by kelliedesigner** [comment]
- use natural light in a real location 
- ensure you have clear descriptions for screenreaders
